lldb/unittests/StackID/StackIDTest.cpp

```diff
@@ -76,25 +76,27 @@ struct MockStackID : StackID {
 };
 
 TEST_F(StackIDTest, StackStackCFAComparison) {
-  auto process = MockProcess(m_target_sp, Listener::MakeListener("dummy"));
```

Comment:
In case you're puzzled by this change: `FixDataAddress` indirectly calls "shared_from_this" on the process class. So it can't be stack allocated.